docworks-cli/src/docworks.js

```diff
@@ -175,6 +176,8 @@ function docworks() {
       .describe('o', 'output file')
       .alias('d', 'dir')
       .describe('d', 'output dir')
+      .alias('i', 'ignore')
+      .describe('i', 'ignores package and does not include it in the output. can be specified multiple times')
```

Comment:
How can we pass multiple packages? (and please add a test for this case)

Reply:
improved documentation